src/style/app.module.css

```diff
@@ -375,17 +385,14 @@
 }
 
 .dropdown {
-  background-color: var(--bs-white) !important;
+  background-color: var(--sortBtn-bg) !important;
   min-width: 150px;
-  border: 1px solid var(--brown-color);
-  color: var(--brown-color) !important;
+  border: 1px solid var(--dark-grey-border);
+  color: var(--dark-grey-font) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`